hardware/CommunicationsPi/lan_client.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+import json
+import requests
+from requests.exceptions import HTTPError
+from utils import get_logger
+
+logging = get_logger("LAN_CLIENT_LOG_FILE")
+
+url = os.environ["LAN_SERVER"]
+
+logging.info('Pinging')
+while True:
+    try:
+        payload = {
+            'key1': 'value1',
+            'key2': 'value2'
+        }
+        logging.info('data: ' + json.dumps(payload))
+        response = requests.post(url, data=payload)
+        response.raise_for_status()
```

Comment:
is the lan_client only for sending data 1 way or will it be checking for responses as well?

Reply:
We would be checking the response status code and response as well just to make sure that we don't discard sensor data before it is logged into the system. If any exception (line 22-25) occurs, we'll have to add handlers in such situations.

Reply:
Ah gotcha sounds good. I will approve this PR, would you mind creating a story to capture adding the exception handlers?

Reply:
Great call. I think it should be a part of the local data store. WIll create a ticket for that.